.config/CredScanSuppressions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "tool": "Credential Scanner",
+  "suppressions": [
+    {
+      "placeholder": "PLACEHOLDER",
+      "_justification": "This is the password used for the TdsServerCertificate.pfx file in the project. It is used for testing purposes only."

```

Comment:
How does this suppression target a specific usage?  What is the meaning of `PLACEHOLDER`?

Reply:
Placeholder is the password in the test file that's being supressed

Reply:
The failure in the internal rolling build is flagging the password argument in this exact line of code:  https://github.com/dotnet/maintenance-packages/blob/57a7b484b596d9971cd7462abf93a191a24d9247/src/System.Data.SqlClient/tests/Tools/TDS/TDS.Servers/TDSServerArguments.cs#L91

I was shown how the aspire repo suppressed similar test lines.

Reply:
Well that's not clear at all from the file contents or comment

Reply:
Not sure what you mean? The _justification says it's a password.